charts/kargo/templates/dex-server/secret.yaml

```diff
@@ -27,10 +27,10 @@ stringData:
       http: 0.0.0.0:5558
 
     oauth2:
-      skipApprovalScreen: {{ .Values.api.oidc.dex.skipApprovalScreen }}
+      skipApprovalScreen: {{ .Values.api.oidc.dex.skipApprovalScreen | default "true" }}
```

Comment:
This prevents invalid configuration if a user blanks out the setting in `values.yaml`.